docs/install.md

```diff
@@ -24,9 +24,12 @@
 
 ### Get source
 
-1. Install git
-2. Get source code:
-
+1. Install git: 
+    ```
+    sudo apt install git
```

Comment:
Please remove this change from PR. This section describes steps for all platforms, not just for Debian/Ubuntu flavors with `apt-get`.

Reply:
This is now done.